care/facility/migrations/0489_migrate_daily_rounds.py

```
@@ -0,0 +1,2626 @@
+            observations_objects = []
+            for observation in observations:
+                try:
+                    pydantic_observation = ObservationSpec(
+                        **observation,
+                        subject_type=selected_questionnaire.subject_type,
+                        data_entered_by_id=daily_round.last_edited_by_id,
+                        created_by_id=daily_round.created_by_id,
+                        updated_by_id=daily_round.last_edited_by_id,
+                    )
+                except Exception as e:
+                    print(e, observation)
```

Ruff (0.8.2)

`print` found

(T201)